redis.conf

```diff
@@ -1762,3 +1762,29 @@ rdb-save-incremental-fsync yes
 # Maximum number of set/hash/zset/list fields that will be processed from
 # the main dictionary scan
 # active-defrag-max-scan-fields 1000
+
+############################### PMEM CONFIG ###############################
+
+# Redis supports four different Memory Allocation Policies:
+#
+# only-dram: use only DRAM - do not use Persistent Memory
+# only-pmem: use only Persistent Memory - do not use DRAM
+# mixed-threshold: use Persistent Memory and DRAM - use threshold described by pmem-threshold
+# mixed-ratio: use Persistent Memory and DRAM - use ratio described by dram-pmem-ratio
+memory-alloc-policy only-dram
+
+# The syntax of dram-pmem-ratio directive is the following:
+#
+# dram-pmem-ratio <dram_value> <pmem_value>
+#
+# Expected proportion of memory placement between DRAM and Persistent Memory.
+# dram_value and pmem_value are values from range 1-INT_MAX
+# In the example below the behaviour will be to:
```

Comment:
behavior

Reply:
what's the relationship between the allocation policy and the dram-pmem-ratio; pmem-threshold? we need have a clear description here. 
1. memory-alloc-policy=only-dram or memory-alloc-policy=only-pmem, no need to se the dram-pmem-ratio
2. memory-alloc-policy=mixed-threshold, static threshold and no need the dram-pmem-ratio, but pmem-threshold is needed. 
3. memory-alloc-policy=mixed-ratio,  has to define the dram-pmem-ratio and no need pmem-threashold some dynamic internal threshold is used.